src/orangepi/ProductionFiles/FinalPiProgram.py

```diff
@@ -23,7 +23,7 @@ class myWebcamVideoStream:
   global testmode, myStrPub, Livemode, RingMode, Aprilmode, Orangepi
   testmode = False
   Livemode = True
-  RingMode = True
+  RingMode = False
```

Comment:
we should look into adding these to the smart dashboard as choosable widget so you can change this when the robot is driving.